packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations.tsx

```diff
@@ -84,7 +84,12 @@ function useHandleDisconnect(fieldName: string, consumerName: string) {
       }
     }
 
-    addFieldRow(`${fieldName}.disconnect`, { id: relation.id });
+    addFieldRow(`${fieldName}.disconnect`, {
+      id: relation.id,
+      api_data: {
```

Comment:
just a style question but I wonder why snake case?

Reply:
just doing too much rust right now :D I'll rename it